### ISSUE TITLE
Package request: hygen

### ISSUE THREAD

**Issue by mainrs** [NixOS/nixpkgs#165640]
**Project description**
`hygen` is a template CLI that allows one to specify project-specific templates that one can invoke to generate boilerplate code.

**Metadata**

* homepage URL: https://github.com/jondot/hygen
* source URL: https://github.com/jondot/hygen
* license: mit
* platforms: unix, linux, darwin

Is it enough to include the package here? https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/node-packages/node-packages.json

**Comment by ModdedGamers** [comment]
> Is it enough to include the package here? https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/node-packages/node-packages.json

Yes, I'm running the generation script now. PR coming in 1-2 hours(hopefully)

**Comment by milgner** [comment]
With the referenced MR closed, I'm gonna give it a try. Looks straightforward to integrate and I'd like to use hygen within my Nix-Flakes-based project, too.
